[Notifications] Clarify plan requirements and instructions (#4131)
cloudflare/cloudflare-docs: content/fundamentals/notifications/create-notifications/configure-webhooks.md

# Configure webhooks

> {{<Aside>}}
> 
> This feature is only available if your account has at least one Professional or higher zone. For more information, see our [plans page](https://www.cloudflare.com/teams-pricing/).
> 
> {{</Aside>}}

Reviewer: ```suggestion
{{<Aside type="note">}}

This feature is only available if your account has at least one zone on a Professional or higher plan. For more information, see our [plans page](https://www.cloudflare.com/plans/).

{{</Aside>}}
```

After configuring the external service you want to connect to, set up webhooks in your Cloudflare dashboard:

> 1. Go to [notifications](https://dash.cloudflare.com/?to=/:account/notifications).

Reviewer: ```suggestion
1. Go to [Notifications](https://dash.cloudflare.com/?to=/:account/notifications) on the Cloudflare dashboard.
```

> 2. Click **Destinations** on the under the **Notifications** header.

Reviewer: ```suggestion
2. Click **Destinations**.
```
